Strip HTML in excerpt field and adjust test data

ulliholtgrave opened the pull request:
### Short description
<!-- Describe this PR in one or two sentences. -->
This PR adds a functionality to strip the HTML entities in the excerpt field and adjust the test data accordingly.

### Resolved issues
<!-- List all issues which should be closed when this PR is merged. -->

Fixes: #1410

timoludwig reviewed (commented):
Thanks, works as expected! :+1:
Is there a reason you didn't use the inbuilt Django functionality?
```
from django.utils.html import strip_tags

excerpt": strip_tags(page_translation.content),
```
I feel like a custom html parser is a bit overkill for this task...
Apart from that: could you add a note to the changelog?

ulliholtgrave commented:
@timoludwig You are right, i just missed this inbuild method and changed it accordingly. I also added a Changelog entry and fixed all the test cases (which was a hell of a job :see_no_evil: )
